packages/core/template_project/app/build.gradle

```diff
@@ -70,11 +70,16 @@ android {
 
         <% if (webManifestUrl) { %>
             // The URL the Web Manifest for the Progressive Web App that the TWA points to. This
-            // is used by Chrome OS to open the Web version of the PWA instead of the TWA, as it
-            // will probably give a better user experience for non-mobile devices.
+            // is used by Chrome OS and Meta Quest to open the Web version of the PWA instead of
+            // the TWA, as it will probably give a better user experience for non-mobile devices.
             resValue "string", "webManifestUrl", '<%= webManifestUrl %>'
         <% } %>
 
+        <% if (fullScopeUrl) { %>
```

Comment:
if fullScopeUrl && isMetaQuest?

Reply:
Why then don't we use `<% if (webManifestUrl && isChromeOSOnly { %>` instead of `<% if (webManifestUrl) { %>` in line 71?

Reply:
And also why then don't we use `<% if (webManifestUrl && isChromeOSOnly) { %>` instead of `<% if (webManifestUrl) { %>` in `packages/core/template_project/app/src/main/AndroidManifest.xml` file?

Reply:
`webManifestUrl` is additional information that can be used by anyone and, nowadays, it's always saved on `init` - it doesn't have to be exclusive to Chrome OS. I think `fullScopeUrl` would fall in the same category - it won't cause any bad side effects to have it. The counter argument would be bloating the APK, but I'd guess it's small enough that it won't have significant impact.